### PR TITLE
Chore: Clean up localfixtures, env, Cypress tests to only have CST

### LINKED ISSUE #2210: Refactor `local_fixtures.json` to remove real agency samples and replace with CST
Edit the `local_fixtures.json` (used to initialize the app when running locally or in CI) to contain only CST sample data.

### LINKED ISSUE #2211: Update Cypress tests to reflect changes to "out of the box" app with only CST
Depends on #2210.

Once the out of the box app with `local_fixtures.json` is set up for CST only, the Cypress tests need to be updated to pass for this configuration.

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #2210 
closes #2211

This work is related to #2209 #2205 and other tickets a part of removing specific real agencies from non-production apps.

## What this PR does
- Remove MST, SacRT, SBMTD objects from fixtures file and `.env.sample` file
- Replace MST object with CST, rename Courtesy Card to Agency Card

## Testing
- Cypress tests are expected to fail
- App should run normally with 1 agency, 4 elig verifiers and look like https://dev-benefits.calitp.org/

## Post-merge tasks
- env.sample, env can be simplified

### CODE REVIEW

**Comment by machikoyasuda** [comment]
@angela-tran Are you able to go through the entire Agency Card flow locally with this fixture file? I'm _not_ currently but it might be because my Elig Server image is messed up.

**Comment by angela-tran** [comment]
> @angela-tran Are you able to go through the entire Agency Card flow locally with this fixture file? I'm _not_ currently but it might be because my Elig Server image is messed up.

I can get through some of it. Here's where I'm at:

- `./bin/reset_db.sh`

Tried to verify Agency Card eligibility, saw an error about `NoneType` with getting a secret.
- add a value for `agency_card_verifier_api_auth_key` in my `.env` file, similar to `courtesy_card_verifier_api_auth_key`

Tried to verify Agency Card eligibility again, got the "Your card information may not have been entered correctly." screen, and saw this in `server` logs:
```
[2024-07-11 16:49:44,352] DEBUG eligibility_server.verify:33 Verify initialized without hashing
2024-07-11 11:49:44 [2024-07-11 16:49:44,447] DEBUG eligibility_server.verify:123 User's types do not contain any of the requested types: ['agency_card']
```
Options to fix:
 - add `agency_card` to the [data file](https://github.com/cal-itp/benefits/blob/main/.devcontainer/server/settings.py#L14) we're using
 - create separate data file in `.devcontainer/server` for Benefits to use instead of looking at eligibility-server's sample data file, and add `agency_card` to it

**Comment by machikoyasuda** [comment]
Yes! I was just about to comment that we'll have to also make a ticket to clean up MST/SBMTD-specific data from Elig Server test files too.

**Comment by machikoyasuda** [comment]
Done here: https://github.com/cal-itp/eligibility-server/pull/484/files

**Comment by machikoyasuda** [comment]
@angela-tran @thekaveman @lalver1 

I decided to do both #2210 #2216 in one PR, because the latter requires the former for it to work, and the former requires the latter for it to pass GitHub tests. I believe once the changes to the server.csv file and associated specs here https://github.com/cal-itp/eligibility-server/pull/484 are merged and deployed, then this PR should then pass.

File: tests/pytest/eligibility/test_forms.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from benefits.eligibility.forms import CSTAgencyCard, MSTCourtesyCard, SBMTDMobilityPass
```
Comment (thekaveman) [inline tests/pytest/eligibility/test_forms.py:1]: I think we should keep these tests, as we do still need the form classes and we can assert that they are set up correctly.

**Review by thekaveman — CHANGES_REQUESTED** [review]
Looking really good so far. One small request.

File: tests/pytest/eligibility/test_forms.py (hunk 1 above)
Comment (machikoyasuda) [reply tests/pytest/eligibility/test_forms.py]: Reverted that last commit: https://github.com/cal-itp/benefits/pull/2214/commits/e8838b21bdbc5f9711c488f4eb291cf220643279

File: benefits/core/migrations/local_fixtures.json [hunk 2]
```diff
@@ -185,101 +145,36 @@
     "model": "core.eligibilityverifier",
     "pk": 3,
     "fields": {
-      "name": "(MST) eligibility server verifier",
+      "name": "(CST) eligibility server verifier",
```
Comment (thekaveman) [inline benefits/core/migrations/local_fixtures.json:148]: (Not for this PR): eventually we should have a new `core.authprovider` instance for this verifier:

```json
{
  "model": "core.authprovider",
  "pk": 3,
  "fields": {
    "sign_out_button_template": "core/includes/button--sign-out--login-gov.html",
    "sign_out_link_template": "core/includes/link--sign-out--login-gov.html",
    "client_name": "calfresh-benefits-oauth-client-name",
    "client_id_secret_name": "auth-provider-client-id",
    "authority": "https://example.com",
    "scope": "verify:calfresh",
    "claim": "calfresh",
    "scheme": "dev-cal-itp_benefits"
  }
},
```

Then update the field here on the `eligibilityverifier`:

```diff
- "auth_provider": 1
+ "auth_provider": 3
```

File: benefits/core/migrations/local_fixtures.json [hunk 3]
```diff
@@ -185,101 +145,36 @@
...
-      "unverified_template": "eligibility/unverified--sbmtd-mobility-pass.html",
-      "help_template": "core/includes/help--sbmtd-mobility-pass.html"
-    }
-  },
-  {
-    "model": "core.eligibilityverifier",
-    "pk": 7,
-    "fields": {
-      "name": "(MST) CalFresh oauth claims via Login.gov",
+      "name": "(CST) CalFresh oauth claims via Login.gov",
```
Comment (thekaveman) [inline benefits/core/migrations/local_fixtures.json:171]: (Not for this PR): eventually we should have a new `core.authprovider` instance for this verifier:

```json
{
  "model": "core.authprovider",
  "pk": 3,
  "fields": {
    "sign_out_button_template": "core/includes/button--sign-out--login-gov.html",
    "sign_out_link_template": "core/includes/link--sign-out--login-gov.html",
    "client_name": "calfresh-benefits-oauth-client-name",
    "client_id_secret_name": "auth-provider-client-id",
    "authority": "https://example.com",
    "scope": "verify:calfresh",
    "claim": "calfresh",
    "scheme": "dev-cal-itp_benefits"
  }
},
```

Then update the field here on the `eligibilityverifier`:

```diff
- "auth_provider": 1
+ "auth_provider": 3
```

**Review by thekaveman — APPROVED** [review]

**Pull request merged**